src/coreclr/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -799,6 +799,9 @@ private ISymbolNode GetHelperFtnUncached(CorInfoHelpFunc ftnNum)
                 case CorInfoHelpFunc.CORINFO_HELP_JIT_REVERSE_PINVOKE_EXIT:
                     id = ReadyToRunHelper.ReversePInvokeExit;
                     break;
+                case CorInfoHelpFunc.CORINFO_HELP_STRCNS_CURRENT_MODULE:
+                    id = ReadyToRunHelper.GetString;
```

Comment:
I do not think it is as simple as this. There needs to be a code to add the current module argument for the helper (it is what `ZapLazyHelperThunk` does in crossgen1.

Reply:
Ah, I thought the `CORINFO_HELP_STRCNS_CURRENT_MODULE` meant we didn't need that. I'll look into adding that lazy thunk.

Reply:
I think that ImportThunk already has a Lazy variant that I think was supposed to cater for lazy strings, among others. Having said that, it hasn't been tested until now so there's a high chance of it being broken in some way.

Reply:
Thanks Tomas - I corrected things to call the lazy thunk and fixed an x64 bug. I'll kick off a CG2 run to make sure we're okay on other architectures.